dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/tools/AbstractJspModC.java

```diff
@@ -1,14 +1,11 @@
 /*******************************************************************************
- * Copyright (c) 1997, 2004 IBM Corporation and others.
+ * Copyright (c) 1997, 2021 IBM Corporation and others.
```

Comment:
I assume 2021 here is used because that is when these changes were first pushed from Evan?

Reply:
correct